gap/utilities.g

```diff
@@ -46,3 +46,68 @@ BindGlobal( "__SIMPLICIAL_ListToCycle",
 );
 
 
+
+#############################################################################
+##
+#F LexicographicallyLeastDyclet . . . . . . .  lexicographically least dyclet
+## 
+## A dyclet represents the orbit of a list under the diherdral group of the
+## list. Find the lexicographically least representative of the dyclet.
+##
+
+# called for lists of sets
+BindGlobal( "__SIMPLICIAL_LexicographicallyLeastDyclet",
+    function(dyclet)
+
+    local n, min_dyclet, k, perm, mirror_dyclet;
+
+    # this test is not conclusive as a list with rationals but
+    # first entry an integer passes this test
+    if not IsHomogeneousList(dyclet) or Length(dyclet) = 0 or
+        not IsInt(dyclet[1]) then
```

Comment:
The comment states that the test given in the implementation is not conclusive. Is this solved by the following if clause? "if Filtered(dyclet,i-> not IsInt(i)) <> [] and Length(dylet)=0 then"

Reply:
It could be
Filtered(dyclet,i-> not IsInt(i)) <> [] or Length(dyclet) =  0